techniques/general/G213.html

```diff
@@ -2,132 +2,118 @@
 <html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
    <head>
       <meta charset="UTF-8" />
-      <title>G213: Provide conventional controls and an application setting for motion activated input</title>
+      <title>G213: ”動き”で操作する入力方法には、従来のコントロールとアプリケーション設定を提供する</title>
```

Comment:
https://waic.jp/docs/WCAG21/Understanding/motion-actuation.html
https://www.w3.org/WAI/WCAG21/Understanding/motion-actuation.html
から、activate を作動に。
```suggestion
      <title>G213: 動きで作動させる入力のために、従来のコントロール及びアプリケーション設定を提供する</title>
```